.gitignore

```diff
@@ -2,6 +2,15 @@
 # - A leading slash means the pattern is anchored at the root.
 # - No leading slash means the pattern matches at any depth.
 
+# Sublime files
+*~
+
+# Random files
+get-pip.py
+setup.py-e
+test/play.py
+v/
```

Comment:
I think the only one that really belongs in .gitignore is `setup.py-e`. The others are not generic build byproducts or anything like that, they are changes that you named, so they should be in your personal .gitignore (you can put one in your home directory I believe).